x-pack/plugins/maps/server/routes.js

```diff
@@ -525,25 +525,23 @@ export async function initRoutes(core, getLicenseId, emsSettings, kbnVersion, lo
       },
     },
     (context, request, response) => {
-      return new Promise((resolve, reject) => {
-        const santizedRange = path.normalize(request.params.range);
-        const fontPath = path.join(__dirname, 'fonts', 'open_sans', `${santizedRange}.pbf`);
-        fs.readFile(fontPath, (error, data) => {
-          if (error) {
-            reject(
-              response.custom({
-                statusCode: 404,
-              })
-            );
-          } else {
-            resolve(
-              response.ok({
-                body: data,
-              })
-            );
-          }
-        });
-      });
+      const range = path.normalize(request.params.range);
+      return range.startsWith('..')
```

Comment:
I don't think this check is enough to ensure that only files within the fonts folder are read. 

Someone could pass in `./../` to traverse up the directory structure. 

The best way to solve this is to resolve the path from range and ensure it is inside your fonts directory.

Reply:
path.normalize handles this case and converts `./../` to `../`. I have added a test case to verify this case.